_includes/main.liquid

```diff
@@ -193,6 +194,7 @@
                                     class="footer-list-item-link">Contract reference</a></li>
                             <li class="footer-list-item"><a href="/chainlink-nodes/"
                                     class="footer-list-item-link">Node operators</a></li>
+                            <li class="footer-list-item"><a href="#" id="dark-toggle" class="footer-list-item-link">Dark Mode</a></li>
```

Comment:
@StephenFluin why put the toggle button at the bottom? Wouldn't it be better to put it at the top of the page ?